index.js

```diff
@@ -6,14 +6,15 @@ function changeTextColor(e) {
   let newText = e.target.parentNode.querySelector("p");
```

Comment:
as soon as you are using e.target.parentNode more than once, just create a variable and use it in 2 places.